### PR TITLE
Fixes for installing R 3.1.2 on osx

### PR DESCRIPTION
**Pull request by gregvonkuster**
I have also tested these changes on Linux and all installs fine.

### CODE REVIEW

File: packages/package_ncurses_5_9/tool_dependencies.xml [hunk 1]
```diff
@@ -20,14 +20,14 @@
...
+                    <environment_variable action="set_to" name="NCURSES_LIB_PATH">$INSTALL_DIR/lib</environment_variable>
                     <environment_variable action="set_to" name="NCURSES_ROOT_PATH">$INSTALL_DIR</environment_variable>
                     <environment_variable action="prepend_to" name="LD_LIBRARY_PATH">$INSTALL_DIR/lib</environment_variable>
                     <environment_variable action="prepend_to" name="DYLD_LIBRARY_PATH">$INSTALL_DIR/lib</environment_variable>
                     <environment_variable action="prepend_to" name="LIBRARY_PATH">$INSTALL_DIR/lib</environment_variable>
-                    <environment_variable name="C_INCLUDE_PATH" action="prepend_to">$INSTALL_DIR/include</environment_variable>
-                    <environment_variable name="CPLUS_INCLUDE_PATH" action="prepend_to">$INSTALL_DIR/include</environment_variable>
+                    <environment_variable name="C_INCLUDE_PATH" action="prepend_to">$INSTALL_DIR/include/ncursesw</environment_variable>
+                    <environment_variable name="CPLUS_INCLUDE_PATH" action="prepend_to">$INSTALL_DIR/include/ncursesw</environment_variable>
```
Comment (bgruening) [inline packages/package_ncurses_5_9/tool_dependencies.xml]: Is this really `ncurses__w__`? I don't have this under Linux? Is this something specific to OS-X?

Comment (gregvonkuster) [inline packages/package_ncurses_5_9/tool_dependencies.xml]: Yes, this is what I found.  Here is is on my Mac:

$ pwd
/Users/gvk/work/ts_install/tool_dependency_dir/ncurses/5.9/iuc/package_ncurses_5_9/77ecbc5b0449/include/ncursesw
MacBook-Pro-2:ncursesw gvk$ ll
total 368
drwxr-xr-x  11 gvk  staff    374 Dec 29 20:25 .
drwxr-xr-x   3 gvk  staff    102 Dec 29 20:25 ..
-rw-r--r--   1 gvk  staff   2891 Dec 29 20:25 eti.h
-rw-r--r--   1 gvk  staff  18005 Dec 29 20:25 form.h
-rw-r--r--   1 gvk  staff  11969 Dec 29 20:25 menu.h
-rw-r--r--   1 gvk  staff  93597 Dec 29 20:25 ncurses.h
-rw-r--r--   1 gvk  staff   3925 Dec 29 20:25 ncurses_dll.h
-rw-r--r--   1 gvk  staff   4072 Dec 29 20:25 panel.h
-rw-r--r--   1 gvk  staff  40299 Dec 29 20:25 term.h
-rw-r--r--   1 gvk  staff   3479 Dec 29 20:25 termcap.h
-rw-r--r--   1 gvk  staff   3109 Dec 29 20:25 unctrl.h

Comment (bgruening) [inline packages/package_ncurses_5_9/tool_dependencies.xml]: Different revision, but it looks like this for me:
```
[galaxy@cn029 ~]$ ls -l /usr/local/tools/ncurses/5.9/iuc/package_ncurses_5_9/5e1760c773ba/include/
insgesamt 4
drwxr-xr-x+ 2 galaxy galaxy 4096  2. Aug 21:13 ncurses
```

This is unfortunate ....

Comment (bgruening) [inline packages/package_ncurses_5_9/tool_dependencies.xml]: Seems to be different packages: https://github.com/openwrt/packages/issues/526 one is with UTF-8 support. Let's figure our how we can compile the other one on your OS-X. There is hopefully a build switch :)

Comment (gregvonkuster) [inline packages/package_ncurses_5_9/tool_dependencies.xml]: Ok, but I'm kinda confused how I got a different package using the same install recipe which downloads the same source.  Hmmm...also I just realized I may not have actually tested these changes on Linux.  I thought I had, but looking again, I think I was using the original recipes on Linux (got confused cause my Linux env is remote).  So, not quite sure how to proceed here...maybe I'll get some sleep first. ;)

**Comment by mvdbeek** [comment]
@gregvonkuster for me it is sufficient to remove all those export statements before ./configure ... perhaps that's a better fix?
This is on the latest dev branch of galaxy, el capitan 10.11.2, gcc installed from the CommandLineTools package, with a clean galaxy instance and the following brew environment:
```
cloog
gcc
gdbm
gettext
git
glib
gmp
gnupg
igv
isl
libffi
libmpc
mpfr
node
openssl
p7zip
pkg-config
postgresql
python
sqlite
unp
wakeonlan
wget
```

**Comment by gregvonkuster** [comment]
@mvdbeek Yes, that's what I discovered as well (you/re referring to the R recipe, right?).  I'm running the dev branch of galaxy, el capitan 10.11.2 as well, and I only have the the following installed via brew:pkg-config, gcc.  Not sure what version of OS X all the export statements actually worked on, but they do not work on El Capitan.

**Comment by mvdbeek** [comment]
Yes! I think the issue with the export statements is that on OS X this somehow fails if any of them is not defined.

**Comment by gregvonkuster** [comment]
Closing since this works for OS X https://toolshed.g2.bx.psu.edu/view/mvdbeek/package_r_3_1_2/174b90e5a7ec

**Pull request closed without merging**